### DISCUSSION TITLE
TypeScript server version

### DISCUSSION THREAD

**Discussion by 12dollar** [aws-samples/aws-saas-factory-ref-solution-serverless-saas#18] [Q&A] [answered]
First of all, 

great documentation and serverless saas example @anubhavynr and team.
Planning to kick off my SaaS on AWS infrastructure I'm curious if there is any good documentation or even workshop for the same solution hereunder using TypeScript on the server-side. I feel keeping programming language on the client- and server-side consistent makes it much easier to maintain/find people to work on.

**Comment by anubhavynr** [comment, marked as answer]
Thanks for the feedback!

Unfortunately, don't have this content in TypeScript at the moment.

**Reply by 12dollar** [reply]
Ok thanks. Any documentation available which could help in migrating from python to TypeScript? A similar project based on TypeScript would already help in kicking this off.

**Reply by anubhavynr** [reply]
Hi @12dollar : see if this helps: https://github.com/aws-samples/aws-saas-factory-eks-saas-workshop/blob/main/services/shared/apps/tenant-management/src/tenants/tenants.service.ts
